github/pulls.go

```diff
@@ -42,7 +42,8 @@ type PullRequest struct {
 	StatusesURL  *string    `json:"statuses_url,omitempty"`
 	DiffURL      *string    `json:"diff_url,omitempty"`
 	PatchURL     *string    `json:"patch_url,omitempty"`
-	Assignee     *User      `json:"assignee,omitempty"` // probably only in webhooks
+	Assignee     *User      `json:"assignee,omitempty"`
+	Assignees    []*User    `json:"assignees,omitempty"`
```

Comment:
@gmlewis According to [GH API](https://developer.github.com/v3/pulls/#labels-assignees-and-milestones), "Every pull request is an issue, but not every issue is a pull request."
That said, I believe all the fields from issues are applicable to pull requests.

Besides that, if you run the API request to this PR (`curl "https://api.github.com/repos/google/go-github/pulls/411"`) you going to see it there:

![image](https://cloud.githubusercontent.com/assets/26548/17338449/73692a76-58b5-11e6-9911-fbfe6f72efa0.png)